Generic Export Indicators Service: added default value to the query (#17998)
demisto/content: Packs/EDL/ReleaseNotes/3_0_4.md

#### Integrations
##### Generic Export Indicators Service

> - Fixed an issue on run *!export-indicators-list-update* command without *query* argument brakes the nginx server.

Reviewer: ```suggestion
- Fixed an issue where the ***export-indicators-list-update*** command would fail when given an empty *query* argument.
```
